src/libs/ReportUtils.ts

```diff
@@ -2817,32 +2844,31 @@ function updateReportPreview(iouReport, reportPreviewAction, isPayRequest = fals
                 type: CONST.REPORT.MESSAGE.TYPE.COMMENT,
             },
         ],
-        childLastMoneyRequestComment: comment || reportPreviewAction.childLastMoneyRequestComment,
-        childMoneyRequestCount: reportPreviewAction.childMoneyRequestCount + (isPayRequest ? 0 : 1),
+        childLastMoneyRequestComment: comment || reportPreviewAction?.childLastMoneyRequestComment,
```

Comment:
```suggestion
        childLastMoneyRequestComment: comment ?? reportPreviewAction?.childLastMoneyRequestComment,
```
Shouldn't be this?

Reply:
no , because in case that comment is `''` then childLastMoneyRequestComment would be an `''` instead of `reportPreviewAction?.childLastMoneyRequestComment` because `??` operator is working only on `undefined` and `null`